e2e/config/playwright-config.ts

```diff
@@ -8,8 +8,8 @@ const config: Config<PlaywrightTestOptions> = {
   testDir,
   outputDir,
   timeout:       process.env.CI ? 600_000 : 300_000,
-  globalTimeout: 600_000,
-  workers:       process.env.CI ? 1 : undefined,
+  globalTimeout: 900_000,
+  workers:       1,
```

Comment:
We cannot run RD in multiples runners, fixing it as a single runner and increased the global execution timeout.